src/net.cpp

```diff
@@ -690,11 +690,16 @@ bool CNode::ReceiveMsgBytes(Span<const uint8_t> msg_bytes, bool& complete)
     return true;
 }
 
+std::string CNode::LogIP(bool log_ip) const
+{
+    return log_ip ? strprintf(" peeraddr=%s", addr.ToStringAddrPort()) : "";
+}
+
```

Comment:
Why not:
```suggestion
std::string CNode::LogIP() const
{
    return fLogIPs ? strprintf(" peeraddr=%s", addr.ToStringAddrPort()) : "";
}
```

Reply:
I think we should avoid accessing the global variable `fLogIPs`. A future refactor could add `m_log_ips` to `PeerManagerImpl` and/or `CConnman`, initialize it from `gArgs` and then drop `fLogIPs`.